migration-engine/json-rpc-api-build/methods/introspect.toml

```diff
@@ -35,3 +35,6 @@ shape = "u32"
 
 [recordShapes.introspectionWarning.fields.message]
 shape = "string"
+
+[recordShapes.introspectionWarning.fields.affected]
+shape = "serde_json::Value"
```

Comment:
The idea with these types was that we would use them to generate both rust and typescript types. That obviously didn't pan out so I'm fine with using something specific like that as a quick solution. We should probably dismantle json-rpc-build and replace it with a macro at some point in the future, if we stick with json-rpc.

Reply:
Thanks for your input. Let's consider this for a possible future PR